doc/BenchmarkingStandard.md

```diff
@@ -247,20 +285,20 @@ Each field type:
 
 | Field | Mandatory | Description |
 | ----- | --------- | ----------- |
-| workarounds | If 'workarounds' in capabilities | List of workarounds as defined in the capabilities file with their current value when the run was made |
 | app_version | Yes | String describing the current application version |
-| std_version | Yes | Integer describing the current version of the standard supported by the application writing the results file |
+| std_version | Yes | String describing the current version of the standard supported by the application writing the results file |
 | date | No | The current date in ISO format |
+| enable_file | Yes | Verbatim copy of the provided enable file |
 | surface_width | No ||
 | surface_height | No ||
-| enable_file | Yes | Verbatim copy of the provided enable file |
+| rendering_backend | No | If the application has multiple rendering backends, specify which one in this free form text field. |
+| start_time | If end_time provided | The time in milliseconds since the system-defined Epoch that the application started initializing. |
+| end_time | No | The time in milliseconds since Epoch that the application started terminating. |
```

Comment:
If start_time is the same as in scene block, then end_time should replace stop_time as well, I think.